models.py

```diff
@@ -0,0 +1,39 @@
+""" Helper module to manage torch vision models """
+from torchvision.models.alexnet import __all__ as all_alexnet_models
+from torchvision.models.vgg import __all__ as all_vgg_models
+from torchvision.models.squeezenet import __all__ as all_squeezenet_models
+from torchvision.models.densenet import __all__ as all_densenet_models
+from torchvision.models.mnasnet import __all__ as all_mnasnet_models
+from torchvision.models.mobilenet import __all__ as all_mobilenet_models
+
+
+# maps the tuple of available model names to the layer from which we want to
+# extract the embedding. Removes the first entry because it the model class
+# not the factory method.
+_all_supported_models_to_layer_mapping = {
+    tuple(all_alexnet_models[1:]): 'features',
+    tuple(all_vgg_models[1:]): 'features',
+    tuple(all_squeezenet_models[1:]): 'features',
+    tuple(all_densenet_models[1:]): 'features',
+    tuple(all_mnasnet_models[1:]): 'layers',
+    tuple(all_mobilenet_models[1:]): 'features',
+}
+
+
+def is_model_supported(name: str) -> bool:
+    return sum([name in model_names for model_names in _all_supported_models_to_layer_mapping]) > 0
```

Comment:
this is very weird, isn't there something like `any`? or have a flatten list of all?

Reply:
yep, thanks for the tip https://github.com/jina-ai/executor-image-torch-encoder/pull/7/commits/1ac07261ac5a2d1de8e657fdaaef10d027545a14

Reply:
This is the same as `return any([name in model_names for model_names in _all_supported_models_to_layer_mapping])`